cypress/support/selectors.js

```diff
@@ -18,6 +18,9 @@ export default {
   LoadStores: 'span[class*=loadAll]',
   MoreItems: 'span[class*=addressListLink]',
   AddressContainer: 'div[class*=addressContainer]',
+  Hours: '.vtex-store-locator-0-x-hoursContainer',
```

Comment:
Here am assuming it is an div but check once before you update
div[class*=hoursContainer]